### PR TITLE
some tiny things

### PR DESCRIPTION
**Pull request by Apricitine**
i changed some of the media queries so that the horizontal layout fits within 700px. this may or may not be good depending on whether or not u guys want to have the extension itself vertically formatted

### CODE REVIEW

**Review by RoblKyogre — CHANGES_REQUESTED** [review]
Seems like times get crunched when viewing a specific day in the calendar (also happens to a lesser extent on the main website tho, to a lesser extent):
image
There also appears to be some clipping on the "Now" period screen:
image
image
(top: this PR | bottom: main website)
Otherwise, looks good to me! (may also request another review to see if we want the extension to be vertical or not)

**Comment by Apricitine** [comment]
woah oopsy i definitely didn't test the calendar page
i tried to fix the clipping but ig not :/

**Comment by Apricitine** [comment]
thats weird
Screenshot (10)
image
mine looks completely normal, if not a little squeezed
is it that our device pixel ratios are different maybe?
ill remove the horizontal scaling on period.current thats probably causing it

**Comment by Apricitine** [comment]
okay i think the problem with the calendar is fixed
i also edited the layout of the calendar a bit

**Comment by mark-crichton** [comment]
apologies for not responding, been moving into college since yesterday so haven't really gotten much of an opportunity to look at this

**Comment by mark-crichton** [comment]
aight i have more time now that it's the weekend, so time to review

Not sure if it's a device pixel ratio thing, as nothing seems to change when adjusting it in devtools
Also the clipping occurs in both firefox and chromium for me:
image image
(ignore weird zoom on chromium, it's ui is far too zoomed in on my device for some reason)
Did actually just now notice this on the live go page as well, so would be fine merging even if it didn't get fixed:
image

Regardless, the smaller device view seems strange now, as the current period is no longer aligned (think it's because you removed `.period.current` from the section in `@media screen and (max-width: 670px)`):
image
imo looks better if it's added back in:
image

File: style.css [hunk 1]
```diff
@@ -642,9 +662,7 @@ option.sr-select-item {
   .period-information {
     display: none;
   }
-  .period.current {
-    left: 0px;
-  }
```
Comment (mark-crichton) [inline style.css:647]: not sure why this was removed, I think it looks better if it was left in

**Review by mark-crichton — CHANGES_REQUESTED** [review]
tbh I think it might be okay to merge apart from this one thing

File: style.css (hunk 1 above)
Comment (Apricitine) [reply style.css:647]: woah college very cool indeed

Comment (Apricitine) [reply style.css:647]: yeah i think everything in this pr that deals with the ui is kinda goofy
i agree with u

**Comment by Apricitine** [comment]
yeah i think this can be closed everything here is breaking and not very beneficial
ill try to get something better working

**Pull request closed without merging**